src/protectedApi_v8/workflow-handler.ts

```diff
@@ -298,3 +298,38 @@ workflowHandlerApi.post('/userWFApplicationFieldsSearch', async (req, res) => {
         )
     }
 })
+
+workflowHandlerApi.post('/v2/userWFApplicationFieldsSearch', async (req, res) => {
```

Comment:
this is older style of implementation we followed... we intercepted all the requests in ui-proxy itself and then called the backend service API from ui-proxy... 
this approach simply adds more load into ui-proxy -- so do use the proxies path for this... 

changes required...
proxies_ts file --> add handler for --> workflowhandler/v2 -- and simply forward the requests to kong... 
whitelist api ---> api path would be --> 
/proxies/v8/workflowhandler/v2/userWFApplicationFieldsSearch